pkg/server/server.go

```diff
@@ -308,3 +315,23 @@ func mapConditions(conditions []metav1.Condition) []*pb.Condition {
 
 	return out
 }
+
+//Until the middleware is done this function will not be able to get the token and will fail
+func (s *applicationServer) Authenticate(ctx context.Context, r *pb.AuthenticateRequest) (*pb.AuthenticateResponse, error) {
```

Comment:
Nit: the convention we have been using is to label the request payload `msg`. `r` is generally reserved for an `http.Request` object.